scss/typography/_base.scss

```diff
@@ -441,3 +433,26 @@ $abbr-underline: 1px dotted $black !default;
     }
   }
 }
+
+/// Calculates the header font-size based on breakpoints.
+///
+/// @param {String} the size of the header to output.
+@mixin typography-header-font-size (
+  $requested-size
+) {
+  // Heading sizes
+  @each $size, $headers in $header-sizes {
+    @include breakpoint($size) {
```

Comment:
This mixin does two things:
**1** - generate headers (or any other selectors) from a font map. It could probably be used by the users.
**2** - generate this set of headers (or any other selectors) for each breakpoint. It is used by Foundation, and maybe some users.

So I think this mixin should be divided in two mixin : `typography-font-sizes` (**1**) and `typography-bp-font-sizes` (**2**).

@kball What do you think ? Do you have better names ?